### PR TITLE
lambda@fileDelete and lambda@fileUpload

### PR DESCRIPTION
**Pull request by ayush0x00**
# Description

Added lambdas for fileDelete and fileUpload

Fixes a part #20 

## Type of change

Please delete options that are not relevant.

- [x] Created new lambdas `@deleteFile` and `@uploadFile`

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: lambdas/file_access_patterns/lambda_for_fileDelete/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "zip -r ../out/lambda_for_fileDelete.zip *"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-gambit/VT-lambdas"
+  },
+  "author": "Lakshya Bansal <https://github.com/lakshya-20>",
```
Comment (danishjamal104) [inline lambdas/file_access_patterns/lambda_for_fileDelete/package.json]: update this.

File: lambdas/file_access_patterns/lambda_for_fileUpload/package.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "zip -r ../out/lambda_for_fileUpload.zip *"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-gambit/VT-lambdas"
+  },
+  "author": "Lakshya Bansal <https://github.com/lakshya-20>",
```
Comment (danishjamal104) [inline lambdas/file_access_patterns/lambda_for_fileUpload/package.json]: here also

**Review by danishjamal104 — CHANGES_REQUESTED** [review]
Rest everything is good

File: lambdas/file_access_patterns/lambda_for_fileDelete/package.json (hunk 1 above)
Comment (ayush0x00) [reply lambdas/file_access_patterns/lambda_for_fileDelete/package.json]: Done

File: lambdas/file_access_patterns/lambda_for_fileUpload/package.json (hunk 2 above)
Comment (ayush0x00) [reply lambdas/file_access_patterns/lambda_for_fileUpload/package.json]: Done

**Review by danishjamal104 — APPROVED** [review]
LGTM

**Review by lakshya-20 — APPROVED** [review]

**Pull request merged**